README.md

```diff
@@ -33,50 +33,69 @@ demonstrate the functionalities of Scalar DB. The Q&A application is a web appli
 - Question Submission
 - Answering Submission
 
-Let's set up the environment for the demonstration so that you can try it out to have a better understanding of the functionalities. We will explain how to develop this application with Scalar DB and what differs to use Scalar DB with and without transaction.
+Let's set up the environment for the demonstration so that you can try it out to have a better understanding of the functionalities. 
+We will explain how to develop this application with Scalar DB and what differs to use Scalar DB with and without transaction.
+
+This document contains different underlying storage/database (such as cassandra, cosmos db) set up and configurations, Please follow any one of them.
 
 ## Set up the environment
```

Comment:
```suggestion
## Install prerequisites
```

Reply:
Thank you for your suggestion.
Document modified accordingly.